leap/leap.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program determines if a year is a leap year.")
+	fmt.Println("Enter the year:")
+	var year int
+	fmt.Scan(&year)
+	if year%400 == 0 {
```

Comment:
Could you try to refactor it in a way that you print each message only once? Smth like this:
```go
if complicated condition {
  fmt.Println(year, "is a leap year.")
} else {
  fmt.Println(year, "is not a leap year.")
}
```